src/TodoList.js

```diff
@@ -1,32 +1,17 @@
 import React from "react";
+import TodoListItem from './TodoListItem';
 
-const todoList = [
-    {
-      id: 1,
-      title: 'Pay Rent',
-    },
-    {
-      id: 2,
-      title: 'Complete Assignment',
-    },
-    {
-      id: 3,
-      title: 'Walk Dog',
-    },
-];
 
-function TodoList() {
-    return (
-        <ul>
-            {todoList.map(function (item) {
-                return (
-                    <li key={item.id}>
-                        {item.title}
-                    </li>
-                )
-            })}
-      </ul>
-    );
+function TodoList({todoList}) {
+  return (
+    <ul>
+      {todoList.map((item) => {
+        return (
+          <TodoListItem key={item.newTodo.id} todo={item.newTodo} />
```

Comment:
Great job mapping through the todoList state - just one little thing the key and the todo props should use the title and id keys instead of newTodo.

These are the keys of the newTodo object

{title: todoTitle, id: Date.now()}

so the todoList should map like this

<TodoListItem key={item.id} todo={item.title} />